test/unit/service/jobRoleServiceTest.ts

```diff
@@ -41,5 +41,39 @@ describe('jobRoleService', () => {
             expect(e.message).to.equal('Failed to get job roles');
         }
     });
+    it('should return 200 and ID of updated job', async () => {
```

Comment:
Missing scenario on the other side of the if, when there is no exception but status code isn't 200